src/bin/benchmarking/uploadResults.mjs

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env node
+import { execSync } from 'child_process';
+import { promises as fs } from 'fs';
+import * as path from 'path';
+
+import { program } from 'commander';
+
+import { mostRecentBenchmarkFilePath } from 'queries/util.mjs';
+import { EC2, EC2ResultsDir } from '../../node_modules/config.mjs';
+
+program
+.requiredOption(
+	'-p, --path <path>',
+	'Path to directory benchmark results'
+);
+
+program.parse(process.argv);
+const options = program.opts();
+
+(async () => {
+	// remove http:// from string
```

Comment:
These things could be precomputed in a file like `src/node_modules/configured.mjs` or so, that would import from `src/node_modules/config.mjs`, with everyone importing from one and/or the other depending on the circumstances.

This way we'd have computation happenning only once and in a single place.